code/game/objects/structures/mannequin.dm

```diff
@@ -42,15 +43,22 @@
 	)
 	/// Assoc list of all item slots (turned to strings) to the items they hold.
 	var/list/worn_items = list()
+	///List of all clothing items the mannequin should be spawning in with on Initialize.
+	var/list/obj/item/clothing/starting_items = list()
 
 /obj/structure/mannequin/Initialize(mapload)
 	. = ..()
 	for(var/slot_flag in slot_flags)
 		worn_items["[slot_flag]"] = null
+		for(var/obj/item/clothing/items as anything in starting_items)
+			items = new items(src)
+			if(items.slot_flags & slot_flag)
+				worn_items["[slot_flag]"] = items
+				continue
```

Comment:
oh, this loop seems wonky, you create the entire set of items for every slot flag
idea:
local list of instances from the paths in starting_items
then you look through that list on each loop, if you successfully put something in a slot (its not full and the slot flags match), remove it from the list
if the list still has starting items even though the loop ended, make a CRASH()